File: lec_03/greet.cpp
#include <iostream>

using namespace std;

void mood(string name) {
    cout << "How are you, " + name + "?" << "\n";     
}


void bye() {
    cout << "See you soon!" << "\n";
}


void greet(string name) {
    cout << "Hello, " + name + "!" << "\n";
    mood(name);
    cout << "It's time to say goodbye..." << "\n";
    bye();
}


int main() {
    string name;
    cin >> name;
    greet(name);
    return 0;
}

File: lec_03/greet.py
def mood(name):
    print("How are you, %s?" % name)


def bye():
    print("See you soon!")


def greet(name):
    print("Hello, %s" % name)
    mood(name)
    print("It's time to say goodbye...")
    bye()


name = str(input())
greet(name)